tensorrt_llm/_torch/pyexecutor/model_engine.py

```diff
@@ -553,7 +566,8 @@ def no_cuda_graph():
                 logger.info(f"Run warmup for batch size={bs}")
                 self.forward(batch,
                              new_tensors_device=None,
-                             resource_manager=resource_manager)
+                             resource_manager=resource_manager,
+                             extra_model_inputs=_create_extra_inputs(bs, 1))
```

Comment:
Why num_tokens = 1 here?

Reply:
That should probably be renamed `num_tokens_per_batch`. This one is just doing decoding request warmups. So 1 token each.

Reply:
Could you add the renaming? It's a simple change but makes it easier to understand.